pkg/convert/testdata/programs/required_providers/pcl/main.pp

```diff
@@ -0,0 +1,24 @@
+package "null" {
+  baseProviderName    = "terraform-provider"
+  baseProviderVersion = "0.3.0"
+  parameterization = {
+    name    = "hashicorp/null"
+    value   = "eyJyZW1vdGUiOnsidXJsIjoicmVnaXN0cnkub3BlbnRvZnUub3JnL2hhc2hpY29ycC9udWxsIiwidmVyc2lvbiI6IjMuMi4zIn19"
+    version = ""
+  }
+}
+
+package "random" {
+  baseProviderName    = "terraform-provider"
+  baseProviderVersion = "0.3.0"
+  parameterization = {
+    name    = "hashicorp/random"
+    value   = "eyJyZW1vdGUiOnsidXJsIjoicmVnaXN0cnkub3BlbnRvZnUub3JnL2hhc2hpY29ycC9yYW5kb20iLCJ2ZXJzaW9uIjoiMy41LjEifX0="
+    version = "~> 3.5.1"
```

Comment:
I believe this still needs to be a resolved version i.e. `3.5.1` instead of a range `~> 3.5.1`

Reply:
saw this after i clicked merge, wish that unresolved comments blocked that...

For now nothing depends on this being "right" so as I implement the actual converting I'll adjust as needed.  It is not so simple to resolve this version without pulling in more deps iirc but I'll investigate and make sure we do the right thing